nav2_smac_planner/src/a_star.cpp

```diff
@@ -375,7 +445,7 @@ bool AStarAlgorithm<NodeT>::createPath(
 template<typename NodeT>
 bool AStarAlgorithm<NodeT>::isGoal(NodePtr & node)
 {
-  return node == getGoal();
+  return _goalsSet.find(node) != _goalsSet.end();
```

Comment:
Can you benchmark the changes before and after your PR to see what the runtime difference between these changes look like?

Reply:
working on this

Reply:
@SteveMacenski  Updated the readme page.